espnet2/tasks/abs_task.py

```diff
@@ -330,44 +394,46 @@ def get_parser(cls) -> configargparse.ArgumentParser:
             default=1,
             help="The number of gradient accumulation",
         )
-
         group.add_argument(
             "--no_forward_run",
             type=str2bool,
             default=False,
             help="Just only iterating data loading without "
             "model forwarding and training",
         )
+        group.add_argument(
+            "--resume",
+            type=str2bool,
+            default=False,
+            help="Enable resuming if checkpoint is existing",
```

Comment:
Now we cannot specify the checkpoint, right?

Reply:
Yes. I concluded that many people wouldn't do and simplicity is more important.